CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+Welcome to the Fedimint project! We're thrilled that you're interested in contributing to our open-source community. Below are some resources to help you get started:
+
+- **[Contributing Guidelines](https://github.com/fedimint/fedimint/blob/master/docs/contributing.md)**: Read our guidelines to understand how you can contribute to Fedimint.
+
+- **[Development Environment Setup Guide](https://github.com/fedimint/fedimint/blob/master/docs/contributing.md)**: Follow this guide to set up your development environment.
+
+If you have any questions, don't hesitate to reach out to us on our [Developer Discord](https://chat.fedimint.org). Our community is friendly and supportive, and we're here to help you succeed.
```

Comment:
Should we have a channel dedicated to new contributors or is general good enough for it?

Reply:
I made one #new-contributors

Reply:
just update that link so they land on that channel specifically then it's good